Include/internal/pycore_pyatomic_ft_wrappers.h

```diff
@@ -61,14 +61,61 @@ extern "C" {
     _Py_atomic_store_uint16_relaxed(&value, new_value)
 #define FT_ATOMIC_STORE_UINT32_RELAXED(value, new_value) \
     _Py_atomic_store_uint32_relaxed(&value, new_value)
+#define FT_ATOMIC_STORE_CHAR_RELEASE(value, new_value) \
+    _Py_atomic_store_char_release(&value, new_value)
+#define FT_ATOMIC_LOAD_CHAR_RELAXED(value) \
+    _Py_atomic_load_char_relaxed(&value)
+#define FT_ATOMIC_STORE_UCHAR_RELEASE(value, new_value) \
+    _Py_atomic_store_uchar_release(&value, new_value)
+#define FT_ATOMIC_LOAD_UCHAR_RELAXED(value) \
+    _Py_atomic_load_uchar_relaxed(&value)
+#define FT_ATOMIC_STORE_SHORT_RELEASE(value, new_value) \
+    _Py_atomic_store_short_release(&value, new_value)
+#define FT_ATOMIC_LOAD_SHORT_RELAXED(value) \
+    _Py_atomic_load_short_relaxed(&value)
+#define FT_ATOMIC_STORE_USHORT_RELEASE(value, new_value) \
+    _Py_atomic_store_ushort_release(&value, new_value)
+#define FT_ATOMIC_LOAD_USHORT_RELAXED(value) \
+    _Py_atomic_load_ushort_relaxed(&value)
+#define FT_ATOMIC_STORE_INT_RELEASE(value, new_value) \
+    _Py_atomic_store_int_release(&value, new_value)
+#define FT_ATOMIC_LOAD_INT_RELAXED(value) \
+    _Py_atomic_load_int_relaxed(&value)
+#define FT_ATOMIC_STORE_UINT_RELEASE(value, new_value) \
+    _Py_atomic_store_uint_release(&value, new_value)
+#define FT_ATOMIC_LOAD_UINT_RELAXED(value) \
+    _Py_atomic_load_uint_relaxed(&value)
+#define FT_ATOMIC_STORE_LONG_RELEASE(value, new_value) \
+    _Py_atomic_store_long_release(&value, new_value)
+#define FT_ATOMIC_LOAD_LONG_RELAXED(value) \
+    _Py_atomic_load_long_relaxed(&value)
+#define FT_ATOMIC_STORE_ULONG(value, new_value) \
+    _Py_atomic_store_ulong(&value, new_value)
+#define FT_ATOMIC_STORE_SSIZE_RELEASE(value, new_value) \
+    _Py_atomic_store_ssize_release(&value, new_value)
```

Comment:
Those two are asymmetric in their declaration (namely, I cannot find the `_Py_atomic_load_ulong_relaxed` nor the `_Py_atomic_load_ssize_relaxed`). Are they somewhere else or just not needed? (or maybe behind a if-guard?)

Reply:
They were already defined:
- [_Py_atomic_load_ulong_relaxed](https://github.com/python/cpython/blob/297f2e093ec95800ae2184330b8408c875523467/Include/cpython/pyatomic.h#L552)
- [_Py_atomic_load_ssize_relaxed](https://github.com/python/cpython/blob/297f2e093ec95800ae2184330b8408c875523467/Include/cpython/pyatomic.h#L358)

Reply:
Huh it's really hard to review when the pieces are not at the correct places (I know that it's better for your PR not to change the ordering in this module but I honestly think we need to do one PR which groups correctly the macros and functions...)

Reply:
Came very close to doing today

Reply:
FTR, I opened #123385 where macros are being used. The code looks better IMO.